doc/questions.md

```diff
@@ -0,0 +1,48 @@
+## Questions
+
+Questions are mechanism that allows user to react on certain situations that happen in installer.
```

Comment:
Please use https://www.deepl.com/write for grammar improvement. My work taken over by the robots, YAYYY!

Reply:
Let's make it clear that an Agama Question is not just any question.

```suggestion
[Questions][] are mechanism that allows user to react on certain situations that happen in installer.

[Questions]: https://opensuse.github.io/agama/dbus/ref-org.opensuse.Agama.Questions1.html
```